src/main/java/org/example/Utility.java

```diff
@@ -0,0 +1,56 @@
+package org.example;
+
+import java.io.*;
+import java.net.Socket;
+import java.time.LocalTime;
+import java.util.Map;
+
+public class Utility {
+
+    public Client addNewClient(int port, int n, LocalTime timeStart, Socket socket) {
+        return new Client(uniqueName(n), port, timeStart, socket);
+    }
+
+    public String uniqueName(int n) {
+        String name = "client-" + n;
+        return name;
+    }
+
+    public void messageAll(String message, Map<Client, PrintWriter> cliens, String name) {
+        cliens.forEach((k, v) -> {
+            v.println(name + ": " + message);
+            v.flush();
+        });
+    }
+
+    public void writeToFile(File file, String word, String name) {
+        BufferedWriter bufferedWriter = null;
+        try {
```

Comment:
in case of exception, buffereWriter will not be closed, you can fix it by finally{} block or using try with resources